Makefile

```diff
@@ -29,6 +29,7 @@ GO_VERSION    = 1.7.3
 GO_BUILD      = go build -i -v
 BASE_PATH     = $(ROOT:/src/github.com/kubernetes-incubator/service-catalog/=)
 export GOPATH = $(BASE_PATH):$(ROOT)/vendor
+GIT_VERSION  ?= $(shell git describe --always --abbrev=40 --dirty)
```

Comment:
Can I suggest this as an alternative?

```
GIT_VERSION  ?= $(shell git describe --tags --always --abbrev=7 --dirty)
```

This results in a shorter version string that looks like the abbreviated (7 digit) format that github normally uses wherever it's ugly or inconvenient to display entire shas. It will also prefer to use a git tag (e.g. semver tag) if one exists. This way if you build from an officially released version of the codebase, the resulting Docker image tag will match the git tag.